packages/applicant/src/utils/postQuestion.ts

```diff
@@ -144,7 +144,7 @@ export default async function postQuestion<B, R>(
       return {
         body: body!,
         values: fieldsStartingWithQuestionIdInBody(cleanedBody, questionId).map(
-          ([key, value]) => value
+          ([_key, value]) => value
```

Comment:
could just do 

```suggestion
          ([, value]) => value
```

although that's a bit harder to read if you're not so familiar with js, to be fair